ts/features/wallet/cobadge/component/BaseCoBadgeCard.tsx

```diff
@@ -126,9 +126,9 @@ const BaseCoBadgeCard: React.FunctionComponent<Props> = (props: Props) => {
           {props.caption && (
             <>
               <View style={{ flexDirection: "row" }}>
-                <H4 weight={"Regular"} testID="caption">
+                <Monospace testID="caption" color={"bluegreyDark"}>
```

Comment:
Could be convenient to have a dedicated component so devs can use it directly?

`<BlurredPan>{pan}</BlurredPan>`

what do you think?

Reply:
Good idea! In thiw way we can have a consistent style within different components!

Added in 77db57f & 77db57f